integr8ly/manifests
# INTLY-3169 -  Update amq online version to be GA image tag version

## Description
The AMQ online CSV already currently uses the GA image, however the versioning does not match the image tag. In addition, the image does not seem to point to the latest 1.1 patch tag according to the [registry](https://access.redhat.com/containers/?tab=overview#/registry.access.redhat.com/amq7/amq-online-1-controller-manager) (Click "View All Tags" -> "1.1" -> Current 1.1 tag points to 1.1-5). 

This pull requests updates the packaging stucture versioning to the image tag version and also pull the 1.2 image.

Associated version bump in the integreatly operator - https://github.com/integr8ly/integreatly-operator/pull/133

## Verification
* Create a operator source using that points to my app registry:
```
apiVersion: operators.coreos.com/v1
kind: OperatorSource
metadata:
  name: integreatly-operators
  namespace: openshift-marketplace
spec:
  authorizationToken: {}
  displayName: Integreatly Operators
  endpoint: 'https://quay.io/cnr'
  publisher: Integreatly Publisher
  registryNamespace: kevfan
  type: appregistry
```
* Browese to the operator hub and verifiy that the AMQ Online version is `1.2.2`
* Install Integreatly using the following Installation.cr as an example:
```
apiVersion: integreatly.org/v1alpha1
kind: Installation
metadata:
  name: example-installation
spec:
  type: workshop
  namespacePrefix: integreatly-
  routingSubdomain: apps.cluster-id.openshiftworkshop.com
  masterUrl: https://console-openshift-console.apps.cluster-id.openshiftworkshop.com
  apiUrl: https://api.cluster-id.openshiftworkshop.com
  selfSignedCerts: true
```
* Verify that the `integreatly-amq-online` namespace is created
* Verify that the `enmasse-operator` pod is created
* Verify that the amq online route is accessible
	* Note: Would need to wait for the user-sso operator to be completed before being able to load  the route
